test/unit/mdc-slider/foundation-keyboard-events.test.js

```diff
@@ -0,0 +1,661 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {assert} from 'chai';
+import td from 'testdouble';
+
+import {TRANSFORM_PROP, setupEventTest as setupTest} from './helpers';
+
+suite('MDCSliderFoundation - keyboard events');
+
+function createFakeEvent(info = {}) {
```

Comment:
Are there places you can identify currently where this would be useful? Otherwise I'm skeptical hesitant to prematurely refactor it.